src/lib/api/manager.ts

```diff
@@ -121,12 +123,26 @@ const fromEstate = (estate: EstateFields, role: RoleType) => {
   return result
 }
 
+function isMax(result: LandQueryResult) {
+  // true if any result length matches the MAX_RESULTS constant
+  return (
+    result.tenantParcels.length >= MAX_RESULTS ||
```

Comment:
Maybe refactor to something like `Object.values(result).some(x => x.length >= MAX_RESULTS)`

Reply:
Afaik every property in the result is an array

Reply:
done ✅